core/language/en-GB/Snippets/GetTiddlyWikiJS.tid

```diff
@@ -0,0 +1,54 @@
+title: $:/language/Snippets/GetTiddlyWikiJS
```

Comment:
I don't understand, why the additional docs is part of `$:/language/` ... same here. IMO it should be part of TiddlyWiki docs. right?

Reply:
Indeed, this is the wrong namespace, but more than that we don't have docs like this within the core. The only docs that are included are the command descriptions for the `--help` command.

Reply:
Hi @Jermolene,

Any preference for the location of this help doc? Would `$:/language/Help/external-js/getcore.tid` be OK?

This doc is meant to be contained in every wiki--so moving into tw5.com edition is not ideal. It guides users on how to acquire core JS based on their particular setups. For example, if a wiki is loaded by TiddlyWiki node.js, a link will be available to help users download the core from their servers, but the "Export TiddlylWiki Core" button will only be enabled if the wiki is not currently rendered with `external-js` template. But I think more importantly, this doc hopes to ensure the version of the core the users acquire will match their own wiki versions.